src/Compilers/Test/Utilities/CSharp/CompilationTestUtils.cs

```diff
@@ -338,16 +338,14 @@ string toDisplayString(SyntaxNode syntaxOpt)
                     {
                         if (trivia.Kind() == SyntaxKind.MultiLineCommentTrivia)
                         {
-                            var expr = getEnclosingExpression(token);
-                            if (expr is null)
-                            {
-                                continue;
-                            }
                             var text = trivia.ToFullString();
                             const string prefix = "/*T:";
                             const string suffix = "*/";
                             if (text.StartsWith(prefix) && text.EndsWith(suffix))
                             {
+                                var expr = getEnclosingExpression(token);
+                                Assert.True(expr != null, $"VerifyTypes could not find a matching expression for annotation '{text}'.");
```

Comment:
:memo: This allows to identify misplaced `/*T:...*/` in tests (which previously were ignored).